test/test.html

```diff
@@ -7,17 +7,11 @@
     <script type="text/javascript" src="../src/vendors/chai/chai.js"></script>
     <script type="text/javascript">
       mocha.setup('bdd');
+
+      window.DONT_INIT_PAGES = true; // disables the initIndex because the err that is thrown there upsets mocha
     </script>
-    <script src="../src/vendors/cryptojs/components/core-min.js"></script>
-    <script src="../src/vendors/cryptojs/components/sha256-min.js"></script>
-    <script src="../src/vendors/cryptojs/components/enc-base64-min.js"></script>
-    <script src="../src/vendors/cryptojs/rollups/hmac-sha512.js"></script>
-    <script src="../src/vendors/bitcore/browser/bundle.js"></script>
-    <script src="../src/js/consts.js"></script>
-    <script src="../src/js/mnemonic.js"></script>
-    <script src="../src/js/util.bitcore.js"></script>
-    <script src="../src/js/util.bitcoin.js"></script>
-    <script src="../src/js/util.js"></script>
```

Comment:
I guess this hadn't been used for a while?
considering it wasn't working anymore ...

can someone confirm?